docs/assets/js/script.js

```diff
@@ -96,6 +101,9 @@ devicon.controller('IconListCtrl', function($scope, $http, $compile) {
     $scope.selectedFontIndex = 0;
     $scope.selectedSvgIndex = 0;
 
+    // Computes miscellaneous data
+    $scope.avgFonts = ($scope.totalFonts / $scope.icons.length).toFixed(2); // Avg number of fonts and SVGs combined
```

Comment:
Returns a string of an integer, instead of a float.
```suggestion
    $scope.avgFonts = ($scope.totalFonts / $scope.icons.length).toFixed(); // Avg number of fonts and SVGs combined
```

Reply:
I actually prefer having a float here, since it better shows that the number is a calculated average imo.

Reply:
hmm, so if there are 17 icon versions and 5 icons you'd prefer that it said `Average icon versions per icon: 3` instead of `Average icon versions per icon: 3.4`?

Reply:
Yeah.
It makes more sense to me, because it sounds like "from 5 icons I have more chance to find 3 versions of it".
I know `3.4` is more precise than `3`, and represent more the average, but I think we can be more exact on this one. 😁

Reply:
Hmm, how about we show the integer, and then we can add the whole number to the title instead?

Reply:
Seems fair to me! 🙂

Reply:
@kilianpaquier Could you change the average number to an integer, and add a title attribute with the float value? That way the whole number is displayed, and on-hover the decimal number will be displayed